src/helpers/getPrettierInstance.test.js

```diff
@@ -22,6 +23,24 @@ test("returns user's project's local prettier instance if it exists", () => {
   );
 });
 
+test("returns global prettier if user's project has no local prettier package", () => {
+  atomLinter.findCached.mockImplementation(() => undefined);
+  const filePath = path.join(__dirname, 'sourceFile.js');
+  const file = { path: filePath, getPath: () => filePath };
+  const editor = createMockTextEditor({ buffer: { file } });
+  const prettierLib = path.join(__dirname, '..', '..', 'tests', 'fixtures', 'prettier.js');
```

Comment:
I'm worried that this is a bit confusing because you're using a local prettier instance as a fake prettier instance to use when there is no local prettier instance (it's even confusing to say in English). I think if just renamed it a bit it would be more clear, something like `fakeGloballyInstalledPrettier`?